go.mod

```diff
@@ -11,6 +11,9 @@ replace github.com/denisenkom/go-mssqldb => github.com/grafana/go-mssqldb v0.0.0
 // It's also present on grafana/loki's go.mod so we'll need till it gets updated.
 replace k8s.io/client-go => k8s.io/client-go v0.18.8
 
+// Loki dependency doesn't currently works with github.com/prometheus/common@v0.21.0. Issue https://github.com/grafana/loki/issues/3659
+replace github.com/prometheus/common => github.com/prometheus/common v0.20.0
```

Comment:
This has to be removed to build after merging with master.
```suggestion
```

Reply:
Yeah I know